pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1445,7 +1451,9 @@ func TestPodEligibleToPreemptOthers(t *testing.T) {
 				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
 				st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
 			}
-			f, err := st.NewFramework(registeredPlugins, "",
+			stopCh := make(chan struct{})
+			defer close(stopCh)
```

Comment:
Also use context.WithCancel() to make the code style consistent?

Reply:
I think it doesn't make a lot of senses to keep codes consistent. This is simple enough.